pyrk/db/tests/test_database.py

```diff
@@ -1,70 +1,67 @@
-from nose.tools import assert_equal, assert_true, assert_false
-
 from pyrk.db import database as d
 
-import unittest
-
+import pytest
 
 def dictfunc():
     return {'t0': 2}
 
+class TestDatabase:
```

Comment:
I think this should be DatabaseTest. 

A "TestDatabase" class sounds to me like a class defining databases made for testing.
However, "DatabaseTest" sounds like a test class for testing databases. 

Though it's arguable we're dealing with the former, I think we're dealing with the latter.

Reply:
I believe "Test" has to come first for pytest to recognize it. When I changed it to DatabaseTest, pytest skipped the class entirely.